src/Raven.Server/Documents/Handlers/Debugging/ServerWideQueriesDebugHandler.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Raven.Server.Documents.Queries;
+using Raven.Server.Routing;
+using Raven.Server.Web;
+
+namespace Raven.Server.Documents.Handlers.Debugging
+{
+    public class ServerWideQueriesDebugHandler : RequestHandler
+    {
+        [RavenAction("/debug/queries/running/live", "GET", AuthorizationStatus.ValidUser)]
```

Comment:
No, this is wrong.
This is server wide, and we have to make sure that this is:
`/admin/debug` and that the permission level is `Operator`

Reply:
We have single endpoint for every user, please have a look at https://github.com/ravendb/ravendb/pull/8436/files/2bb1368e4f0037cc356ebf5798659055073ce222#diff-e1eadb52a3455ec6c3d3be2636d09b4eR16  we filter to allowed databases. So you get only allowed db in this websocket. 

Is it ok then?

Reply:
yes